Makefile

```
@@ -410,6 +410,47 @@ recipe-health-check: ## Verifies $(HEALTH_DOC_PATH) is up to date with the recip
 	fi; \
 	echo "$(HEALTH_DOC_PATH) is up to date"
 
+# Path of the committed nodewright tuning-status doc. Regenerated by
+# `make tuning-docs`, checked-fresh by `make tuning-check`. Hermetic (no network).
+TUNING_DOC_PATH := docs/integrator/components/nodewright.md
+
+.PHONY: tuning-docs
+tuning-docs: ## Regenerates the auto-generated tuning-status table in $(TUNING_DOC_PATH) from the recipe catalog (hermetic, no network)
```

checkmake (0.3.2)

[warning] Target body for "tuning-docs" exceeds allowed length of 5 lines (23).

(maxbodylength)